pkgs/by-name/tt/tts-mod-vault/package.nix

```
@@ -17,25 +18,21 @@ flutter329.buildFlutterApplication rec {
     hash = "sha256-BTs+4QeyVJeg415uiNXww8twQwUInHfB8voWJjeVs20=";
   };
 
-  icon = fetchurl {
-    url = "https://github.com/markomijic/TTS-Mod-Vault/raw/9a8eec2b009ee4b653863fa3d553a309a2d07431/assets/icon/tts_mod_vault_icon.png";
-    hash = "sha256-xFQpVciuBdOxHCdK4IrskPd0gPeb3x5wklYLlR9vvnY=";
-  };
-
   pubspecLock = lib.importJSON ./pubspec.lock.json;
 
-  desktopItem = makeDesktopItem {
+  desktopItems = [ (makeDesktopItem {
     name = "tts-mod-vault";
     exec = "tts-mod-vault";
-    icon = icon;
+    icon = tts-mod-vault;
```

```suggestion
    icon = "tts-mod-vault";
```
don't remember if i missed quotes, but you did, so be careful with what you write